core/arch/interp.c

```diff
@@ -2022,6 +2033,12 @@ bb_process_syscall(dcontext_t *dcontext, build_bb_t *bb)
         }
     }
 #ifdef WINDOWS
+    if (instr_get_opcode(bb->instr) == OP_int) {
+        if (instr_get_interrupt_number(bb->instr) == 0x2e) {
+            /* Special handling of int 2e when it is not the syscall method. */
```

Comment:
So the assumption is that syscall is seen first, and is the primary method?  So that should be checked in check_syscall_method, right?  B/c we won't handle the other order (right?)

Reply:
yes.
To be more precise, "int" is the second method. Wow64 can be the first method as well.